pkg/csi/service/service.go

```diff
@@ -110,40 +111,24 @@ func (s *service) BeforeServe(
 		cfg *cnsconfig.Config
 	)
 
-	cfg, err = common.GetConfig(ctx)
+	// Initialize CO utility in Nodes
+	commonco.ContainerOrchestratorUtility, err = commonco.GetContainerOrchestratorInterface(ctx, common.Kubernetes, clusterFlavor, COInitParams)
 	if err != nil {
-		log.Errorf("failed to read config. Error: %+v", err)
+		log.Errorf("Failed to create CO agnostic interface. Error: %v", err)
```

Comment:
Failed ==> `failed`

Reply:
It's just a log right?

Reply:
Error log messages can start with an upper case character. Only error messages need a lower case starting character.